### PR TITLE
Add some new examples in chapter5/5.5

### PR DESCRIPTION
**Pull request by Phoenix8215**
## Overview
Hi kalfazed,i have added some new examples in chapter5/5.5.And i have already demonstrated that the network output results between Pytorch and TRT almost the same:-) please review it, 3Q~

### What should reviewers check
i will provide some pictures from the output results(one from Pytorch, another is from TRT)
## How to check
- pooling
图片
图片

- upsample
图片
图片

- deconv
图片
图片

- concat
图片
图片

- elementwise & const
图片
图片

- reduce
图片
图片

- slice
图片
图片

### CODE REVIEW

File: chapter5-tensorrt-api-basics/5.5-build-model/models/weights/sample_concat.weights [hunk 1]
```diff
@@ -0,0 +1,5 @@
+4
```
Comment (kalfazed) [inline chapter5-tensorrt-api-basics/5.5-build-model/models/weights/sample_concat.weights:1]: If all these weights can be reproduced from python code, can you remove all these weights?

**Review by kalfazed — CHANGES_REQUESTED** [review]
Thank you for the contribution and sorry for the late response. Please check the comments above and make some changes? For the other parts, it looks good to me.

**Comment by Phoenix8215** [comment]
Hi @kalfazed, 

I have made the requested changes by adding dynamic parameters to the `build_concat` and `build_slice` functions. Additionally, I have removed the unnecessary weights as per your suggestion.

Could you please review the changes again?

Thank you!
🥰

**Comment by Phoenix8215** [comment]
and some results for my test🥰(actually the same)
img_v3_02c7_c0fe94fe-6dfb-462e-ac1d-7eb36e34d8dg
img_v3_02c7_ed7267d5-a4e5-47cb-90c5-e4c81c275c8g

**Review by kalfazed — APPROVED** [review]
LGTM! It's good to merge now. Thank you for the contribution again!

**Comment by Phoenix8215** [comment]
Hi kalfazed,

Thank you for the approval! Unfortunately, I do not have write access to this repository, so I am unable to merge this pull request myself. Could you merge it for me?

Thank you!🥰

**Pull request merged**